portfolio/src/main/webapp/script.js

```diff
@@ -252,24 +260,77 @@ function createListElement(comment) {
   return listElem;
 }
 
+/**
+ * Formats upvote and downvote buttons for each comment. Sets
+ * each button to pressed or unpressed based on data about
+ * whether the current user has pressed either received
+ * from server. 
+ */
 function formatCommentVoteButtons(comment, thisCommentDiv) {
   const upvoteText = document.createElement("p");
   upvoteText.className = "vote-buttons";
   upvoteText.id = `${comment["id"]}-up`;
   upvoteText.innerText = comment["upvotes"];
+
+  /**
```

Comment:
nit: for implementation comments use block style comments (`/* ... */` or` //...`) https://google.github.io/styleguide/javaguide.html#s4.8.6-comments
What you have here (`/** ... */`) are Javadoc comments that are used by function/class headers for documentation generation.